.github/workflows/examples.yml

```diff
@@ -43,12 +43,21 @@ jobs:
 
     - name: Clean up conda environments
       run: |
-        conda env list |                        # list (env name, env path) pairs
-        cut -d' ' -f1 |                         # extract env names
-        grep "^mlflow-[a-z0-9]\{32\}" |         # filter envs created by mlflow
-        xargs -L 1 conda remove --all --yes -n  # pass env names to conda remove
-        conda env list
-        conda clean --all --yes
+        mlflow_envs=$(
```

Comment:
Maybe we can move this to a script, just to avoid any confusion with the python yaml string embedding?

Reply:
Done 👍